index.html

```diff
@@ -1071,7 +1071,7 @@ <h2>
         feedback.
       </p>
       <pre class="idl">
-        [Exposed=Window]
+        [Exposed=Window, SecureContext]
```

Comment:
Not sure this one makes sense either, to be honest. Half of the API is exposed in a regular context, and taking this interface away would probably break things.